ibis/backends/polars/compiler.py

```diff
@@ -1157,3 +1157,11 @@ def execute_view(op, *, ctx: pl.SQLContext, **kw):
 @translate.register(ops.SelfReference)
 def execute_self_reference(op, **kw):
     return translate(op.table, **kw)
+
+
+@translate.register(ops.CountDistinctStar)
+def execute_count_distinct_star(op, **kw):
+    arg = pl.struct(*op.arg.schema.names)
+    if (where := op.where) is not None:
+        arg = arg.filter(translate(where, **kw))
```

Comment:
IMO the use of the `:=` doesn't add a benefit here (attribute access is cheap, no reason to only evaluate it once) and makes the code harder to read. Definitely opinion based though, up to you.

```suggestion
    if op.where is not None:
        arg = arg.filter(translate(op.where, **kw))
```

Reply:
SGTM!